db/seeds.rb

```diff
@@ -149,7 +149,9 @@ def random_record_for_org(org, klass)
   user = User.create(
     email: user_data[:email],
     password: 'password!',
-    password_confirmation: 'password!'
+    password_confirmation: 'password!',
+    organization: user_data[:organization], # Any reason not to set this?
+    name: user_data[:email].split('@').first # Any reason not to set this?
```

Comment:
Was there a reason why we weren't ensuring that an organization and name were in place for users?

Reply:
Once upon a time we needed organizations for bank users.  We changed to a role-based access for all users.   Super users and partners wouldn't have had an organization from the get-go.        As for names,  I think all the bank users in production have names,  but not all the partner users do.   Does that help?

Reply:
Ahhhh I see, that does help thank you! I'll get rid of these two additions and keep that in mind moving forward.